packages/api/src/platforms/vtex/resolvers/product.ts

```diff
@@ -16,84 +18,82 @@ const getPath = (link: string, id: string) => `/${getSlug(link, id)}/p`
+} = {
+  productID: ({ itemId }) => itemId,
+  name: ({ isVariantOf, name }) => name ?? isVariantOf.productName,
+  slug: ({ isVariantOf: { linkText }, itemId }) => getSlug(linkText, itemId),
   description: ({ isVariantOf: { description } }) => description,
-  seo: ({ isVariantOf: { name, description } }) => ({
-    title: name,
+  seo: ({ isVariantOf: { description, productName } }) => ({
+    title: productName,
     description,
   }),
   brand: ({ isVariantOf: { brand } }) => ({ name: brand }),
-  breadcrumbList: ({ isVariantOf: { categoryTrees, name, link }, id }) => ({
-    itemListElement: [
-      ...categoryTrees.reverse().map(({ categoryNames }, index) => ({
-        name: categoryNames[categoryNames.length - 1],
-        item: `/${categoryNames
-          .map((categoryName) => slugify(categoryName))
-          .join('/')}`,
-        position: index + 1,
-      })),
-      {
-        name,
-        item: getPath(link, id),
-        position: categoryTrees.length + 1,
-      },
-    ],
-    numberOfItems: categoryTrees.length,
-  }),
-  image: ({ isVariantOf, images }) =>
-    (
-      nonEmptyArray(images) ??
-      nonEmptyArray(isVariantOf.images) ?? [DEFAULT_IMAGE]
-    ).map(({ name, value }) => ({
-      alternateName: name ?? '',
-      url: value.replace('vteximg.com.br', 'vtexassets.com'),
-    })),
-  sku: ({ id }) => id,
-  gtin: ({ reference }) => reference ?? '',
+  breadcrumbList: ({
+    isVariantOf: { categories, productName, linkText },
+    itemId,
+  }) => {
+    return {
+      itemListElement: [
+        ...categories.reverse().map((categoryPath, index) => {
+          const categoryNames = categoryPath.split('/')
+
+          return {
+            name: categoryNames[categoryNames.length - 2],
+            item: categoryPath.toLowerCase(),
+            position: index + 1,
+          }
+        }),
+        {
+          name: productName,
+          item: getPath(linkText, itemId),
+          position: categories.length + 1,
+        },
+      ],
+      numberOfItems: categories.length,
+    }
+  },
+  image: ({ images }) =>
+    (nonEmptyArray(images) ?? [DEFAULT_IMAGE]).map(
+      ({ imageUrl, imageText }) => ({
+        alternateName: imageText ?? '',
+        url: imageUrl.replace('vteximg.com.br', 'vtexassets.com'),
+      })
+    ),
+  sku: ({ itemId }) => itemId,
+  gtin: ({ referenceId }) => referenceId[0]?.Value ?? '',
   review: () => [],
   aggregateRating: () => ({}),
-  offers: async (product, _, ctx) => {
+  offers: (product, _, ctx): { items: Item[]; product: Root } => {
     const {
-      loaders: { simulationLoader },
       storage: { channel },
     } = ctx
 
-    const { id, policies } = product
+    const { itemId, sellers } = product
 
-    const sellers = policies.find(
-      (policy) => policy.id === channel.salesChannel
-    )?.sellers
+    const sellerFromChannel = sellers.find(
+      (seller) => seller.sellerId === channel.salesChannel
+    )
```

Comment:
Do we need to check this? Since the trade-policy is already a facet? @hiagolcm @thalytafabrine

Reply:
I didn't understand the relationship between the sellerId and the salesChannel. 

maybe this doc can help you to understand the difference between then

https://help.vtex.com/pt/tutorial/qual-a-diferenca-entre-politica-comercial-afiliado-e-seller--frequentlyAskedQuestions_729

All the sellers that we return should be shown on the client-side with no filter. In theory, you should not worry about that.